### PR TITLE
Writing_Good_Commits: Removed the word "the"

### PR DESCRIPTION
**Pull request by tamj0rd2**
Removed "the" from lines 26-28 to improve readability.

Fixes https://github.com/coala-analyzer/documentation/issues/67

### CODE REVIEW

File: Developers/Writing_Good_Commits.rst [hunk 1]
```diff
@@ -23,9 +23,9 @@ How to Write Good Commit Messages
 
 A commit message consists of 3 parts:
```
Comment (Adrianzatreanu) [inline Developers/Writing_Good_Commits.rst]: please period after commit body in commit message

Comment (Adrianzatreanu) [inline Developers/Writing_Good_Commits.rst]: also, try holding a shortlog limit of 50 chars as else github wraps commit messages badly and its unreadable. thanks!

Comment (Adrianzatreanu) [inline Developers/Writing_Good_Commits.rst]: you can remove the ``.rst``

**Comment by Adrianzatreanu** [comment]
@tamj0rd2 what you are doing is changing the Pull Request title :D what i want you to do is change the commit title, which is different. in order to do that, simply type ``git commit --amend``, which will let you edit your last commit's message, change it, save it, and ``git push --force myfork`` :) with force this time.

**Comment by tamj0rd2** [comment]
@Adrianzatreanu Thank you, I've done that now :) Do I need to do another PR?

**Comment by Adrianzatreanu** [comment]
you have changed to past tense. please keep imperative and change to ``Remove the word "the"`` thx

**Comment by tamj0rd2** [comment]
@Adrianzatreanu Thanks, I've updated that now too

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by tamj0rd2** [comment]
@rultor merge

**Comment by Asnelchristian** [comment]
@rultor merge

**Comment by Makman2** [comment]
@tamj0rd2 please don't merge yourself. It shouldn't be possible, but it seems rultor has some issues currently. Please wait/ask until some maintainer does that :+1: :)

**Comment by tamj0rd2** [comment]
@Makman2 sorry!

**Comment by Adrianzatreanu** [comment]
moved to another PR

**Pull request closed without merging**